clustered/client/src/test/java/org/ehcache/clustered/client/internal/PassthroughLeasedConnectionService.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ehcache.clustered.client.internal;
+
+import com.tc.classloader.OverrideService;
+import org.terracotta.connection.Connection;
+import org.terracotta.connection.ConnectionException;
+import org.terracotta.connection.ConnectionFactory;
+import org.terracotta.connection.entity.Entity;
+import org.terracotta.connection.entity.EntityRef;
+import org.terracotta.exception.EntityNotProvidedException;
+import org.terracotta.lease.connection.LeasedConnection;
+import org.terracotta.lease.connection.LeasedConnectionService;
+
+import java.io.IOException;
+import java.net.URI;
+import java.util.Properties;
+
+@OverrideService("org.terracotta.lease.connection.LeasedConnectionServiceImpl")
+public class PassthroughLeasedConnectionService implements LeasedConnectionService {
```

Comment:
Why is this here and not in `terracotta-platform` or `passthrough`?

Note: I was very lazy and did not go back into the code before asking

Reply:
You are right this should go in passthrough

Reply:
Raised https://github.com/Terracotta-OSS/tc-passthrough-testing/pull/160